src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -112,6 +112,8 @@ const SignedChatScreen = () => {
         style={styles.chatContainer}
         data={updatedChats}
         inverted={true}
+        windowSize={10}
+        maxToRenderPerBatch={5}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `FlatList` component in `SignedChatScreen.tsx` are good for performance. By setting `windowSize={10}` and `maxToRenderPerBatch={5}`, you're controlling the number of items rendered at a time, which can help reduce memory usage and improve scrolling performance on devices with limited resources. However, please ensure that these settings provide a smooth user experience across all target devices, as too small a window size or batch size could lead to noticeable loading during fast scrolling.

```diff
+         windowSize={10}
+         maxToRenderPerBatch={5}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->